JobConfig/pionbeam/PiBeamResampler.fcl

```diff
@@ -0,0 +1,79 @@
+#
```

Comment:
I don't see the advantage of putting these scripts in a new directory for this prodution.  POT_pion will be more understandable in beam, PiBeamResampler in pileup.

Reply:
I can move them to more appropriate locations...I just wanted to keep my work separate while I was debugging